modules/openapi-generator/src/main/resources/python/model_oneof.mustache

```diff
@@ -111,6 +111,7 @@ class {{classname}}({{#parent}}{{{.}}}{{/parent}}{{^parent}}BaseModel{{/parent}}
         instance = cls.model_construct()
         {{#isNullable}}
         if json_str is None:
+            instance.actual_instance = None
```

Comment:
I have to add ` = None` here, because now `actual_instance` doesn't have default value. It's kinda crazy that pydantic allows `model_construct` call without filling required fields. They are just missing in runtime then

Reply:
@b0g3r: after reading the 2 `.mustache` changes: would that work for you if, instead of adding `None` as another possible value if `isNullable`, you keep the previous construct with `Optional` and `= None` is `isNullable`?

I haven't tested but I feel this could actually help:

* This change `instance.actual_instance = None` would probably not be necessary
* The typing would be a bit clearer (I think)